libraries/video/shared/helpers.js

```diff
@@ -1,5 +1,19 @@
 import { videoKey } from '../constants/constants.js'
 
 export function getExternalVideoEventName(eventName) {
+  if (!eventName) {
+    return '';
+  }
   return videoKey + eventName.replace(/^./, eventName[0].toUpperCase());
```

Comment:
Here are we assuming that all video players have the event names have the prefix `video` ? 
Should it be configurable with the default value `video`?
Refer: https://github.com/prebid/Prebid.js/blob/master/libraries/video/constants/constants.js#L1

Reply:
@musikele the library code has very limited functionality, looks good to me but I have one doubt (mentioned in the above comment)
@karimMourra please check the above comment.

Reply:
@pm-harshad-mane these events are part of the module's API. When video players surface events, they surface them using the following constants https://github.com/prebid/Prebid.js/blob/master/libraries/video/constants/events.js and we append the `video` portion. It keeps the char count lower. 
This is already documented https://docs.prebid.org/prebid-video/video-module.html